TrainingJava/src/House/ClientHouseBuilder.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package House;
+
+import java.util.Scanner;
+
+/**
+ *
+ * @author andreslietti
+ */
+
+// In this class the house is created requesting data from a client. 
+public class ClientHouseBuilder implements HouseBuilder {
+    private House house;
+    
+    
+    public ClientHouseBuilder(){
+        house = new House();
+    }
+    
+    @Override
+    public void buildStructure() {
+        System.out.println("Choose a structure: ");
+        String choice;
+        Scanner scan = new Scanner (System.in); 
+        choice = scan.nextLine (); 
+        house.setStructure(choice);
+    }
+
+    @Override
+    public void buildRoofTyoe() {
+        System.out.println("Choose type of roof: ");
+        String choice;
+        Scanner scan = new Scanner (System.in);
+       choice = scan.nextLine (); 
+        house.setRoofType(choice);
+    }
+
+    @Override
+    public void NumberOfFloors() {
+        System.out.println("Choose Number of Floors: ");
+        int choice;
+        Scanner scan = new Scanner (System.in); 
+        choice = scan.nextInt(); 
+        house.setNumberOfFloors(choice);
+    }
+
+    @Override
+    public void buildOutdoorType() {
+        System.out.println("Choose a type of Outdoors: ");
+        String choice;
+        Scanner scan = new Scanner (System.in); 
+        choice = scan.nextLine (); 
+        house.setOutdoorType(choice);
+    }
+
+    @Override
+    public void FurnishedHouse() {
+        System.out.println("Do you want furnished? ");
+        String choice;
+        Scanner scan = new Scanner (System.in); 
+        choice = scan.nextLine (); 
+        if (choice.contains("yes")) {
```

Comment:
Couple of concerns here:
* Every time you like to compare strings, the constant should be first, in this case `"yes".equals(choice)` to avoid null pointer exceptions ( perhaps choice is null).
* try to avoid the use of contains() method, use equals() or equalsIgnoreCase()